index.js

```diff
@@ -25,25 +25,25 @@ module.exports = {
         on: 'change',
       },
       label: 'Select one',
+      options: [
+        { name: 'Michaelangelo',
+          value: 'mike',
+        },
+        { name: 'Leonardo',
+          value: 'leo',
+        },
+        { name: 'Raphael',
+          value: 'ralph',
+        },
+        { name: 'Donatello',
+          value: 'don',
+        },
+      ],
       type: 'select',
       settings: {
         empty: true,
-        options: [
-          { name: 'Michaelangelo',
-            value: 'mike',
-          },
-          { name: 'Leonardo',
-            value: 'leo',
-          },
-          { name: 'Raphael',
-            value: 'ralph',
-          },
-          { name: 'Donatello',
-            value: 'don',
-          },
-        ],
       },
     },
   },
-  html: '<label for="{{select.id}}">{{select.label}}</label><select id="{{select.id}}" name="{{select.name}}">{% for name in select.settings.options %}<option value="{{name.value}}" {% if name.value == select.value %}selected{% endif %}>{{name.name}}</option>{% endfor %}</select></label>',
+  html: '<label for="{{select.id}}">{{select.label}}</label><select id="{{select.id}}" name="{{select.name}}">{% for name in select.options %}<option value="{{name.value}}" {% if name.value == select.value %}selected{% endif %}>{{name.name}}</option>{% endfor %}</select></label>',
```

Comment:
Can this be `for option in select.options` instead of `for name in select.options`? It's only `name` in your sample content